cgogn/core/cmap/map_base.h

```diff
@@ -109,8 +109,10 @@ class MapBase : public MapBaseData
 	 */
 	inline void clear_and_remove_attributes()
 	{
+		this->topology_.remove_marker_attributes();
 		this->topology_.clear_chunk_arrays();
 
+		boundary_marker_ = topology_.add_marker_attribute();
```

Comment:
Et bien figure-toi que dans les benchmarks, avant de recourir à des unique_ptr et à faire des delete/new sur la map entre chaque itération, j'utilisais clear_and_remove_attributes. Et je me suis rendu compte que les performances empiraient d'une itération à l'autre. Cette modification résout ce problème de performance.